### PR TITLE
Nachtrag zu Provider umbau

### PR DESCRIPTION
**Pull request by Moetez16**

### CODE REVIEW

File: grails-app/services/de/laser/ProviderService.groovy [hunk 1]
```diff
@@ -465,4 +465,16 @@ class ProviderService {
...
     }
+
+    Set<Long> getCurrentProviderIdsOfProviders(Org context) {
+        Set<Long> result = ProviderRole.executeQuery("select p.id from ProviderRole pr join pr.provider as p where pr.sub in (select sub from OrgRole where org = :context and roleType in (:roleTypes))",
+                [context: context, roleTypes: [RDStore.OR_SUBSCRIPTION_CONSORTIA, RDStore.OR_SUBSCRIBER_CONS, RDStore.OR_SUBSCRIBER]])
+        result
+    }
+
+    Set<Provider> getCurrentProvidersOfProviders(Org context) {
+        Set<Provider> result = ProviderRole.executeQuery("select p from ProviderRole pr join pr.provider as p where pr.sub in (select sub from OrgRole where org = :context and roleType in (:roleTypes))",
```
Comment (croqueGrec09) [inline grails-app/services/de/laser/ProviderService.groovy:476]: pr.subscription

File: grails-app/services/de/laser/VendorService.groovy [hunk 2]
```diff
@@ -615,4 +615,10 @@ class VendorService {
 
     }
 
+    Set<Long> getCurrentVendorIdsOfVendors(Org context) {
```
Comment (croqueGrec09) [inline grails-app/services/de/laser/VendorService.groovy:618]: gteCurrentVendorIds

**Review by croqueGrec09 — COMMENTED** [review]
remarks

**Comment by Moetez16** [comment]
remarks done

**Review by croqueGrec09 — APPROVED** [review]
approved

**Pull request merged**